enumerable.rb

```diff
@@ -0,0 +1,112 @@
+module Enumerable
+  def my_each
+    i = 0
+    return enum unless block_given?
+
+    while to_a.length > i
+      yield(to_a[i])
+      i += 1
+    end
+    self
+  end
+
+  def my_each_with_index
```

Comment:
- [ ] `my_each_with_index` returns an Enumerator if no block is given. Currently, its throwing the error below.
![image](https://user-images.githubusercontent.com/64360974/113575554-b7b5ca00-9626-11eb-9737-cca24b0a18c1.png)